adapter/src/main/java/ar/edu/undec/adapter/service/boostrap/CursoConfig.java

```diff
@@ -0,0 +1,16 @@
+package ar.edu.undec.adapter.service.boostrap;
+
+import ar.edu.undec.adapter.data.repository.RegistrarCursoRepository;
+import curso.input.RegistrarCursoInput;
+import curso.usecase.RegistrarCursoUseCase;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class CursoConfig {
+
+    @Bean
+    public RegistrarCursoInput registrarCurso(RegistrarCursoRepository registrarCursoRepository) {
```

Comment:
Recibir como argumento la Interfaz (para ser OpenClosed)
```suggestion
    public RegistrarCursoInput registrarCurso(RegistrarCursoOutput registrarCursoRepository) {
```